contents/blog/how-to-measure-product-market-fit.md

```diff
@@ -0,0 +1,332 @@
+| <span className="text-green text-lg">✔</span> Easy to track                                                              | <span className="text-red text-lg">✖</span> Not a reliable sign of product-market fit on its own                         |
+| <span className="text-green text-lg">✔</span> Useful when you're pre-revenue                                             | <span className="text-red text-lg">✖</span> Spikes in awareness and user growth can be misleading                        |
+| <span className="text-green text-lg">✔</span> Easy to compare with other companies                                       | <span className="text-red text-lg">✖</span> Not useful for companies with low-profile brands – e.g. enterprise products  |
+| <span className="text-green text-lg">✔</span> Positive feedback and growth is highly motivating                          | <span className="text-red text-lg">✖</span> It's tempting (and easy) to rationalize negative word-of-mouth               |
+
+## Indicator #2: User feedback & PMF Survey
+
+Like word-of-mouth, _consistently_ positive user feedback is an indicator of product-market fit. There's a decent chance you're onto a winner when people tell you they can't live without your product.
+
+To this end, the PMF Survey is a twist on the classic Net Promoter Score, but it's designed _specifically_ for finding product-market fit, and creating a feedback loop for user feedback.
+
+Created by entrepreneur Sean Ellis, the core question (among others) it asks is:
+
+“How would you feel if you could no longer use [ProductName]?”
+
+- a) Very disappointed
+- b) Somewhat disappointed
+- c) Not disappointed
+
+You want to learn everything you possibly can about people who answer "very disappointed". Organize a call and talk to these users. Ask open-ended, probing questions to understand why they love your product. Do the same for users in the "somewhat disappointed" cohort.
+
+Based on his research of 100+ startups, Ellis believes 40% answering "very disappointed" is a strong signal of product-market fit. The more responses you get, the more reliable the signal. Ellis recommends a minimum of 30:
+
+> "In my experience, a minimum of 30 responses is needed before the survey becomes directionally useful. At 100+ responses I am much more confident in the results." – Sean Ellis[^2]
+
+An open research project run by Hiten Shah[^3], co-founder of KISSmetrics, used the PMF Survey on 731 Slack users, the results showed:
+
+- 51% responded they would be very disappointed if they couldn't use Slack
+- These users believed it increased productivity and improved collaboration
+- All groups said video conferencing was a must-have addition
+
+It's no coincidence Slack has since added video conferencing.
+
+### A PMF Survey case study
```

Comment:
We should link to the superhuman post here. It's probably the most famous blog on the topic, and people would find it interesting to read